api/services/elastigroup/azure/spotVms/paths/vmProtection.yaml

```diff
@@ -53,8 +54,9 @@ delete:
   tags:
     - Elastigroup Azure Spot VMs
   parameters:
+    - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: path
+    - in: query
```

Comment:
wrong change.
the URL is https://api.spotinst.io/azure/compute/group/{groupId}/vm/{**vmName**}/protection